# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/clifford/spiralsynthmodular/nodef_val.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./SpiralSound/Plugins/Widgets/Fl_VU_Meter.cxx.org	Mon Jan 20 12:35:00 2003
+++ ./SpiralSound/Plugins/Widgets/Fl_VU_Meter.cxx	Mon Jan 20 12:35:42 2003
@@ -2,7 +2,7 @@
 #include "Fl_VU_Meter.h"
 #include <FL/fl_draw.H>
 
-Fl_VU_Meter::Fl_VU_Meter (int x, int y, int w, int h, const char *l = 0) :
+Fl_VU_Meter::Fl_VU_Meter (int x, int y, int w, int h, const char *l ) :
 Fl_Progress (x, y, w, h, l) {
 }
 
--- ./SpiralSound/Plugins/Widgets/Fl_VU_Meter.h.org	Mon Jan 20 12:35:05 2003
+++ ./SpiralSound/Plugins/Widgets/Fl_VU_Meter.h	Mon Jan 20 12:36:08 2003
@@ -5,7 +5,7 @@
 
 class Fl_VU_Meter : public Fl_Progress {
   public:
-    FL_EXPORT Fl_VU_Meter (int x, int y, int w, int h, const char *l = 0);
+    FL_EXPORT Fl_VU_Meter (int x, int y, int w, int h, const char *l );
   protected:
     FL_EXPORT virtual void draw();
 };
